k4EDM4hep2LcioConv/src/k4Lcio2EDM4hepConv.cpp

```diff
@@ -589,11 +614,13 @@ namespace LCIO2EDM4hepConv {
       }
     }
     // Filling all the OneToMany and OnToOne Relations and creating the AssociationCollections.
+    auto calocontr = createCaloHitContributions(typeMapping.simCaloHits, typeMapping.mcParticles);
     resolveRelations(typeMapping);
     auto assoCollVec = createAssociations(typeMapping, LCRelations);
 
     podio::Frame event;
     // Now everything is done and we simply populate a Frame
+    event.put(std::move(calocontr), "CaloHitContribution");
```

Comment:
We should use a less generic name here to avoid clashes with other collection names.